combat-trainer.lic

```diff
@@ -1183,7 +1183,12 @@ class AbilityProcess
         next unless !timer || (Time.now - timer).to_i > 30
         game_state.cooldown_timers[name] = Time.now if activate_khri?(@kneel_khri, name)
       end
-    @barb_buffs.each{|name| if Flags["ap-#{name}-expired"] then Flags.reset("ap-#{name}-expired"); start_barb_buff(name); end}
+    @barb_buffs.each do |name| 
```

Comment:
Nice change. I need to start getting into this frame of mind. Ruby is really sweet like that.